nx/source/services/lbl.c

```diff
@@ -45,14 +77,26 @@ Result lblSwitchBacklightOff(u64 fade_time) {
     return _lblCmdInU64NoOut(fade_time, 7);
 }
 
-Result lblSetCurrentBrightnessSetting(float brightness) {
-    return serviceDispatchIn(&g_lblSrv, 2, brightness);
+Result lblGetBacklightSwitchStatus(bool *out_value) {
+    u32 tmpout=0;
+    Result rc = serviceDispatchOut(&g_lblSrv, 8, tmpout);
+    if (R_SUCCEEDED(rc) && out_value) *out_value = tmpout & 0xff;
```

Comment:
Is u32->bool really correct?

Reply:
You were right.
I'll add an enum.
0 disabled, 1 enabled, 2, enabling, 3 disabling